### DISCUSSION TITLE
📔【计算机网络】TCP 三次握手与四次挥手

### DISCUSSION THREAD

**Discussion by imageslr** [imageslr/blog#84] [Announcements]
https://imageslr.github.io/2020/07/07/tcp-shake-wave.html 

 Software Elon Z

**Comment by midasism** [comment]
总结的很棒

**Comment by xiaoxiang10086** [comment]
如果已经建立了连接，但是客户端出现故障了怎么办？
每隔 75 分钟发送一个探测报文段

---
修正：
这里的应该不是每隔75分钟，而是每隔75秒吧?
net.ipv4.tcp_keepalive_intvl：单位秒，表示两次探测报文发送的时间间隔，默认为75。

**Comment by imageslr** [comment]
@xiaoxiang10086 写错了，谢谢指出。
